docs/src/main/asciidoc/datasource.adoc

```diff
@@ -259,12 +259,11 @@ For more information about configuring JDBC, see <<jdbc-url,JDBC URL format refe
 [[other-databases]]
 ===== Custom databases and drivers
 
-If you need to connect to a database for which Quarkus does not provide an extension with the JDBC driver,
-you can use a custom driver instead. For example, if you are using the OpenTelemetry JDBC driver in your project.
+If Quarkus does not provide a JDBC extension for your database, use a custom driver, such as OpenTelemetry JDBC.
```

Comment:
This is kind of awkward, since OpenTelemetry has nothing to do with a specific database; it's a JDBC driver wrapper.

```suggestion
If Quarkus does not provide a JDBC extension for your database, or if you need to use a different JDBC driver (for example for OpenTelemetry), you can use configure the JDBC driver explicitly.
```

Reply:
Hey, good point. Thank you. Now, it will be even better.